### PR TITLE
Feature/37

### PR DESCRIPTION
**Pull request by GabrielBueno0511**
create pr for Gabriel Bueno

### CODE REVIEW

File: backend/app/Models/User.php [hunk 1]
```diff
@@ -4,13 +4,16 @@
 
 // use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Tymon\JWTAuth\Contracts\JWTSubject;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
 use Laravel\Sanctum\HasApiTokens;
 
-class User extends Authenticatable
```
Comment (GabrielBueno0511) [inline backend/app/Models/User.php:11]: alter

File: backend/composer.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
         "guzzlehttp/guzzle": "^7.2",
         "laravel/framework": "^10.0",
         "laravel/sanctum": "^3.2",
-        "laravel/tinker": "^2.8"
```
Comment (GabrielBueno0511) [inline backend/composer.json:12]: duplicate

File: backend/config/auth.php [hunk 3]
```diff
@@ -14,7 +14,7 @@
     */
 
     'defaults' => [
-        'guard' => 'web',
```
Comment (GabrielBueno0511) [inline backend/config/auth.php:17]: alter from api

File: backend/database/migrations/2014_10_12_000000_create_companies_table.php [hunk 4]
```diff
@@ -11,22 +11,22 @@
      */
     public function up(): void
     {
-        Schema::create('failed_jobs', function (Blueprint $table) {
```
Comment (GabrielBueno0511) [inline backend/database/migrations/2014_10_12_000000_create_companies_table.php:14]: bug

File: backend/database/migrations/2014_10_12_000000_create_companies_table.php [hunk 5]
```diff
@@ -11,22 +11,22 @@
...
+        Schema::create('companies', function (Blueprint $table) {
             $table->id();
-            $table->string('uuid')->unique();
-            $table->text('connection');
-            $table->text('queue');
-            $table->longText('payload');
-            $table->longText('exception');
-            $table->timestamp('failed_at')->useCurrent();
```
Comment (GabrielBueno0511) [inline backend/database/migrations/2014_10_12_000000_create_companies_table.php:21]: bugs

File: backend/database/migrations/2014_10_12_000000_create_companies_table.php [hunk 6]
```diff
@@ -11,22 +11,22 @@
...
         });
     }
-
+    
     /**
      * Reverse the migrations.
      */
     public function down(): void
     {
-        Schema::dropIfExists('failed_jobs');
```
Comment (GabrielBueno0511) [inline backend/database/migrations/2014_10_12_000000_create_companies_table.php:30]: bug

File: backend/database/migrations/2023_05_04_134428_create_users_table.php [hunk 7]
```diff
@@ -15,9 +15,8 @@ public function up(): void
             $table->id();
             $table->string('name');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
```
Comment (GabrielBueno0511) [inline backend/database/migrations/2023_05_04_134428_create_users_table.php:18]: alter

File: backend/database/migrations/2023_05_04_134428_create_users_table.php [hunk 8]
```diff
@@ -15,9 +15,8 @@ public function up(): void
...
             $table->string('password');
-            $table->rememberToken();
```
Comment (GabrielBueno0511) [inline backend/database/migrations/2023_05_04_134428_create_users_table.php:20]: alter

File: backend/database/migrations/2023_05_04_135142_create_contacts_table.php [hunk 9]
```diff
@@ -11,10 +11,12 @@
      */
     public function up(): void
     {
-        Schema::create('password_reset_tokens', function (Blueprint $table) {
-            $table->string('email')->primary();
-            $table->string('token');
-            $table->timestamp('created_at')->nullable();
```
Comment (GabrielBueno0511) [inline backend/database/migrations/2023_05_04_135142_create_contacts_table.php:17]: I don't know, Rodolfo's thing

File: backend/database/migrations/2023_05_04_135142_create_contacts_table.php [hunk 10]
```diff
@@ -23,6 +25,6 @@ public function up(): void
      */
     public function down(): void
     {
-        Schema::dropIfExists('password_reset_tokens');
```
Comment (GabrielBueno0511) [inline backend/database/migrations/2023_05_04_135142_create_contacts_table.php:26]: bug

Comment (GabrielBueno0511) [inline backend/database/migrations/2023_05_04_135142_create_contacts_table.php:26]: bug

File: backend/routes/api.php [hunk 11]
```diff
@@ -14,6 +16,26 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
```
Comment (GabrielBueno0511) [inline backend/routes/api.php:18]: bugs

**Review by iTzRodz — APPROVED** [review]

**Pull request merged**